File: 프로그래머스/0/120899. 가장 큰 수 찾기/가장 큰 수 찾기.java
class Solution {
    public int[] solution(int[] array) {
        int[] answer = new int[2];
        int n = array[0];
        int max = 0;
        for(int i = 1; i < array.length; i++) {
            
            if(n < array[i]) {
                n = array[i];
                max = i;
            }
        }
        answer[0] = n;
        answer[1] = max;
        return answer;
    }
}

File: 프로그래머스/0/120899. 가장 큰 수 찾기/가장 큰 수 찾기.js
function solution(array) {
    var answer = [array[0], 0];
    
    for (let i = 0; i < array.length; i++) {
        if (array[i] > answer[0]) {
            answer[0] = array[i];
            answer[1] = i;
        }
        
    }
    return answer;
}